app/views/dashboard/_discovery_widget_host.html.erb

```diff
@@ -0,0 +1,23 @@
+<td>
+  <% if host.created_at > 1.day.ago
+       status_glyph = 'glyphicon-plus-sign'
+       status_message = _('New in the last 24 hours')
+       status_color = '#89A54E'
+     elsif host.last_report < 7.days.ago
+       status_glyph = 'glyphicon-exclamation-sign'
+       status_message = _('Not reported in more than 7 days')
+       status_color = '#AA4643'
+     else
+       status_glyph = 'glyphicon-ok-sign'
+       status_message = _('Reported in the last 7 days')
+       status_color = '#4572A7'
+     end
+  %>
+  <span class="glyphicon <%= status_glyph %>"
+      style="color: <%= status_color %>"
+      title="<%= status_message %>"/>
+</td>
```

Comment:
I was wondering if we can put the icon in the same table cell as the name. Or maybe making its own cell not to enlarge when there is plenty of space. In extreme cases (very wide monitors) icons can be little bit far away from the name which is IMHO an UX issue and does not look nice.